src/data/links.yaml

```diff
@@ -947,3 +947,31 @@
   - news
   tags: ["news"]
   description: Aggregations of climate change and action articles from various major news sources.
+
+- title: Green credit cad
+  url: https://www.bunq.com/link/upgrade-greencard
```

Comment:
Have you had any experience with this? My Dutch friend has had some struggles with them doing unexpected and shitty things, like upgrading to premium without warning and threatening to charge a cancellation fee to downgrade again. 

Secondly, I don't like this message "Just by using it, you create a greener world. Zero effort needed." Every needs to do more than have "one tree planted per 100 spent." That's some greenwashing nonsense right there.

Thirdly, I question the value in general. Premium card is €8/m so that is €96, and the green card is €99. The extra €3 does not seem to be spent on anything, so you're paying a little more for... nothing. 1 tree for every €100 spent means they are paying €0.10-0.35 for every 100 you spend. If you spend €2,000 in a month that's 20 trees. That's not nothing, but paying $99 a year for a bank which seems bloody expensive just so you can have 240 trees a year doesn't seem like something I want to advertise. 

What do you think? Is the bank so amazing we should look past all that?

Reply:
I get your point. I've put it here as I believed it's better than nothing ;) 
As for your questions - 
I have that account since 2017. I had no problems at all so far. I agree though, that the card things are kinda pricey. Even the standard 'premium' one. On the other hand it provides you with an access the the bank's API ;) 
And yeah, the amount of trees they plant is really low at the end of the day, unless you are a big spender, which is probably producing more waste than you should.
The good thing about the bank itself IMHO is that it is independent and in a certain way outside of this bankster world. Not totally sure, but I think they follow the sustainable banking thing and dont invest in environmentally bad businesses.
Your call :)

Reply:
Great, I'll take it.